concourse/pipeline.yml

```diff
@@ -286,10 +315,13 @@ templates:
       RESPONSE_OPERATIONS_UI_PORT: '80'
       SAMPLE_SERVICE_HOST: rm-sample-service-concourse-latest.apps.devtest.onsclofo.uk
       SAMPLE_SERVICE_PORT: '80'
+      SECURE_MESSAGE_URL: 'http://ras-secure-message-ci.apps.devtest.onsclofo.uk'
```

Comment:
Does this need to be `http://ras-secure-message-concourse-latest.apps.devtest.onsclofo.uk` instead?